packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -135,6 +179,20 @@ async function checkCustomizedPort(component: string, componentRoot: string, che
   */
 }
 
+function onDidStartTaskHandler(event: vscode.TaskStartEvent): void {
+  const taskId = event.execution.task?.definition?.teamsfxTaskId;
+  if (taskId !== undefined) {
+    taskStartEventEmitter.fire(taskId as string);
+  }
+}
+
+function onDidEndTaskHandler(event: vscode.TaskEndEvent): void {
+  const taskId = event.execution.task?.definition?.teamsfxTaskId;
+  if (taskId !== undefined && trackedTasks.has(taskId as string)) {
```

Comment:
Seems missing `trackedTasks.add()` beforewards.

Reply:
Good catch. Added `trackedTasks.add` and `trackedTasks.delete` whenever end event is fired.